matrix/main.py

```diff
@@ -132,6 +132,12 @@ def setup(matrix, args=None):
     parser.add_argument("-k", "--keep-models", action="store_true",
                         default=False, help="Keep the created per-test models "
                                             "for further inspection")
+    parser.add_argument("-C", "--cloud", default=None,
+                        help="Cloud to deploy models to. Required when this "
+                        "could be ambiguous, as in the case of our JaaS "
+                        "product, which is a shared controllers that can "
+                        "connect to multiple clouds, "
+                        "JaaS.")
```

Comment:
I want to automagic this param, but I don't think that there's a great way of doing so.

With JaaS, you have to manually specify a cloud. If you don't, and you have creds for multiple clouds, we simply don't have a way of knowing what cloud you meant.